library/pom.xml

```diff
@@ -372,9 +377,39 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
```

Comment:
If it can property maybe you we can also just have in all the pom.xml like
<ckeditor-plugins>${ckeditor-plugins} <new plugin data></ckeditor-plugins>

I don't know if that works or not though. ;)